sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationClient.java

```diff
@@ -176,7 +176,7 @@ public final class ServiceBusAdministrationClient {
      */
```

Comment:
Do you have any tests for these changes?

Reply:
In case  we miss something and a regression occurs.

Reply:
This is the reason I think the broken change actual got through. We don't have integration tests for the sync client. Our integration tests cover only the async client. I tried to put together a sync integration client test suite last night, but couldn't get it done. So, to start I thought, let me fix the problem and them add the tests immediately after